packages/web-client/app/services/token-to-usd.ts

```diff
@@ -88,7 +88,18 @@ export default class TokenToUsd extends Service {
     return this.#registeredHelpers.size > 0;
   }
 
-  toUsdFrom(symbol: UsdConvertibleSymbol, amount: BN): number | undefined {
+  toUsdFrom(
+    symbol: UsdConvertibleSymbol | 'DAI' | 'CARD',
+    amount: BN
+  ): number | undefined {
+    if (symbol === 'DAI') {
+      symbol = 'DAI.CPXD';
+    }
+
+    if (symbol === 'CARD') {
+      symbol = 'CARD.CPXD';
+    }
+
```

Comment:
The way we're doing the usd conversion is a bit convoluted right now. We accept `.CPXD` here, but in `layer2-chain` we remove it. I'll clean it in a followup PR.

Reply:
Thanks! merging this